config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  get 'users/new'
```

Comment:
brakuje sciezek do 'create` i `show`. generalnie w przypadku kiedy dla danego zasobu chcesz posiadac caly standardowy zestaw sciezek, to zamiast definiowania pojedynczo mozesz zrobic `resources :users`. ten helper wygeneruje Ci sciezki do wszystkich akcji

Reply:
Poprawione